src/tests/system/tests/test_ipa_trusts.py

```diff
@@ -60,3 +64,67 @@ def test_ipa_trusts__lookup_group_without_sid(ipa: IPA, trusted: GenericADProvid
     status = ipa.sssctl.domain_status(trusted.domain, online=True)
     assert "online status: offline" not in status.stdout.lower(), "AD domain went offline!"
     assert "online status: online" in status.stdout.lower(), "AD domain was not online!"
+
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopologyGroup.IPATrust)
+def test_ipa_trusts__aduser_membership_after_HBAC(ipa: IPA, trusted: GenericADProvider):
+    """
+    :title: Membership update of the AD-user after it's IPA-group is a member of a HBAC rule
+    :description: ADuser's ipa-group membership misses after ipa-group is added to a HBAC rule
+    :setup:
+        1. Create a trusted AD-user
+        2. Create IPA external group and add a AD user as it's member
+        3. Create IPA posix group and add ipa external group as a member
+    :steps:
+        1. Lookup AD user
+        2. Create HBAC rule for all-host-category,
+        3. Add IPA-posix-group in that HBAC rule
+        4. Clear user cache of that user only
+        5. Lookup AD user
+    :expectedresults:
+        1. User is found and is a member of ipa_group_A group
+        2. HBAC rule created
+        3. ipa_group_A group is added to HBAC rule
+        4. User cache expired for that user only
+        5. User is found and is a member of ipa_group_A group
+    :customerscenario: True
```

Comment:
Is this a customer scenario?

Reply:
Update the field, sssd doesn't have this customer scenario.